tests/cli/workflow/taxprofiler/test_cli_taxprofiler_config_case.py

```diff
@@ -9,37 +9,83 @@
 
 from cg.cli.workflow.taxprofiler.base import config_case
 from cg.constants import EXIT_SUCCESS
+from cg.constants.constants import FileFormat
+from cg.io.controller import ReadFile
 from cg.models.cg_config import CGConfig
+from cg.models.taxprofiler.taxprofiler import TaxprofilerParameters, TaxprofilerSample
 
 
-def test_without_options(cli_runner: CliRunner, taxprofiler_context: CGConfig):
-    """Test command without options."""
+def test_defaults(
```

Comment:
I would be in favour of improving the test names for this module

Reply:
done